### PR TITLE
Improve pt_BR translation

### PR DESCRIPTION
**Pull request by mileo**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Technically seems correct :+1:, although I don't know about brazilian itself.

Regards.

File: sale_commission/i18n/pt_BR.po [hunk 1]
```diff
@@ -139,129 +131,126 @@ msgid "Annual"
...
+#: model:ir.model.fields,field_description:sale_commission.field_sale_agent_retention_id
+#: field:sale.agent,retention_id:0
+msgid "Applied retention"
+msgstr "Imposto retido"
 
 #. module: sale_commission
+#: model:ir.model.fields,field_description:sale_commission.field_sale_agent_employee_id
 #: field:sale.agent,employee_id:0
 msgid "Associated Employee"
 msgstr "Empregado associado"
```
Comment (renatonlima) [inline sale_commission/i18n/pt_BR.po:165]: Maybe the translation term could be in CamelCase like original term

Comment (pedrobaeza) [inline sale_commission/i18n/pt_BR.po:165]: Spanish ortographic rules says that it has not to be in caps, but I don't know for brazilian.

Comment (luanaduarte) [inline sale_commission/i18n/pt_BR.po:165]: Translation updated according to the original term, maintaining the terms which used Came|Case and also the ones which didn't use.

File: sale_commission/i18n/pt_BR.po [hunk 2]
```diff
@@ -485,342 +510,348 @@ msgid "Quarterly"
...
 
 #. module: sale_commission
-#: model:ir.model,name:sale_commission.model_recalculate_commission_wizard
-msgid "recalculate.commission.wizard"
-msgstr "recalculate.commission.wizard"
+#: field:invoice.line.agent,invoice_date:0
+#: model:ir.model.fields,field_description:sale_commission.field_invoice_line_agent_invoice_date
+#: model:ir.model.fields,field_description:sale_commission.field_res_partner_agent_type
+#: field:res.partner.agent,type:0
+msgid "unknown"
```
Comment (bwrsandman) [inline sale_commission/i18n/pt_BR.po]: This is usually a sign that your module contains a field which doesn't have a string parameter.

It is better to add a string to these fields:
- invoice.line.agent,invoice_date
- sale_commission.field_invoice_line_agent_invoice_date
- sale_commission.field_res_partner_agent_type
- res.partner.agent,type

File: sale_commission/i18n/pt_BR.po [hunk 3]
```diff
@@ -485,342 +510,348 @@ msgid "Quarterly"
...
 
 #. module: sale_commission
-#: field:settlement,total:0
-#: field:settlement.agent,total:0
-msgid "Total"
-msgstr "Total"
+#: code:_description:0
+#: model:ir.model,name:sale_commission.model_settlement_line
+#, python-format
+msgid "settlement.line"
```
Comment (bwrsandman) [inline sale_commission/i18n/pt_BR.po]: This usually means that your model doesn't have a `_description` and can lead to bad looking chatter messages.

I suggest you add a `_description` to `sale_commission.model_settlement_line` such as `_description = "Settlement Line`

File: sale_commission/i18n/pt_BR.po [hunk 4]
```diff
@@ -485,342 +510,348 @@ msgid "Quarterly"
...
 
 #. module: sale_commission
-#: field:settlement.line,invoice_line_id:0
-msgid "Settled invoice line"
-msgstr "Linha da fatura liquidada"
+#: code:_description:0
+#: model:ir.model,name:sale_commission.model_settlement_agent
+#, python-format
+msgid "settlement.agent"
+msgstr "settlement.agent"
```
Comment (bwrsandman) [inline sale_commission/i18n/pt_BR.po]: This usually means that your model doesn't have a `_description` and can lead to bad looking chatter messages.

I suggest you add a `_description` to `sale_commission.model_settlement_agent` such as `_description = "Settlement Agent`

**Comment by bwrsandman** [comment]
I started pointing out technical issues with the modules, but I realise this is a i18n fix.

It looks :+1:

**Comment by guewen** [comment]
This PR needs a rebase. It displays nothing.

**Comment by mileo** [comment]
Next week i will fix it and send again.

Thanks

Luis Felipe Miléo
KMEE
+55 35 8876-3663
+55 21 4063-8872
mileo@kmee.com.br
kmee.com.br
Em 22/08/2014 10:42, "Guewen Baconnier" notifications@github.com escreveu:

**Comment by mileo** [comment]
=]

Luis Felipe Miléo
KMEE
+55 35 8876-3663
+55 21 4063-8872
mileo@kmee.com.br
kmee.com.br
Em 02/09/2014 07:03, "omar7r" notifications@github.com escreveu:

**Pull request merged**